### PR TITLE
[script.speedtester@matrix] 1.1.1+matrix.1

### PR DESCRIPTION
**Pull request by dagwieers**
### Add-on details:

- **General**
  - Add-on name: Speed Tester
  - Add-on ID: script.speedtester
  - Version number: 1.1.1+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/add-ons/script.speedtester
  
This add-on tests your Internet bandwith and reports both download and upload speeds and latency from within Kodi.

Use this if you are experiencing streaming issues and you suspect your network to be the cause.

### Description of changes:

v1.1.1 (2020-10-10)
- Add localization

v1.1.0 (2020-09-26)
- Move add-on to Add-ons project
- Fix typos and structure of standard output

v1.0.1 (2019-08-06)
- Initial release
    

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: script.speedtester/resources/lib/addon.py [hunk 1]
```diff
@@ -0,0 +1,921 @@
...
+        from urllib2 import urlopen, Request, HTTPError, URLError
+        from urlparse import parse_qs, urlparse
+    except ImportError:  # Python 2.5 and older
+        # pylint: disable=ungrouped-imports
+        from cgi import parse_qs  # pylint: disable=deprecated-method
+        from httplib import HTTPConnection, HTTPSConnection
+        from urllib.parse import urlparse
+        from urllib2 import urlopen, Request, HTTPError, URLError
+
+from xbmc import Monitor, translatePath
```
Comment (enen92) [inline script.speedtester/resources/lib/addon.py:60]: This needs to come from xbmcvfs in matrix and above

File: script.speedtester/resources/lib/utils.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```
Comment (enen92) [inline script.speedtester/resources/lib/utils.py:2]: Looks like you have 2 licenses in the add-on but only refer the Apache license on add-on.xml

Comment (dagwieers) [reply script.speedtester/resources/lib/utils.py:2]: I can relicense, no problem.

File: script.speedtester/resources/lib/addon.py (hunk 1 above)
Comment (dagwieers) [reply script.speedtester/resources/lib/addon.py:60]: Hmm, I was sure we fixed that. It appears it was badly merged.

**Pull request closed without merging**